frontend/src/libs/components/modal/modal.tsx

```diff
@@ -12,12 +12,13 @@ import styles from './styles.module.scss';
 type Properties = {
   children: React.ReactNode;
   title: string;
+  showCrossIcon?: boolean;
 };
 
 const Modal: React.ForwardRefRenderFunction<
   HTMLDialogElement | null,
   Properties
-> = ({ children, title }, reference) => {
+> = ({ children, title, showCrossIcon = true }, reference) => {
```

Comment:
Please follow acceptance criteria

Reply:
@v0ldemar01 what do you mean? it`s like i should delete default falue, or change it naming to isCorrsIconShown?

Reply:
I think its the second one

`B19. Boolean variables start with a prefix that can be answered with "yes"`